app/Commands/LogCommand.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Commands;
+
+use App\InitializesCommands;
+use App\Shell\Docker;
+use Illuminate\Support\Str;
+use LaravelZero\Framework\Commands\Command;
+
+class LogCommand extends Command
+{
+    use InitializesCommands;
+
+    protected $signature = 'logs {containerId?}';
+    protected $description = 'Display container logs.';
+
+    public function handle(): void
+    {
+        $this->initializeCommand();
+
+        $container = $this->argument('containerId');
+
+        if (! $container) {
+            $this->error("Please insert a valid container ID.\n");
+
+            return;
+        }
+
+        $this->logs($container);
+    }
+
+    public function logs(string $container): void
+    {
+        if (Str::contains($container, ' -')) {
+            $container = Str::before($container, ' -');
```

Comment:
What's this covering?

Reply:
Good question.
When I created the command I copied startCommand class and I forgot to delete that rows.
Is still necessary that rows in start and stop commands?

Reply:
I honestly don't remember what it was for :)

Reply:
me too🤣 but now it is ready to merge.